MAUI/Cartesian-Charts/Errorbar-Chart.md

```diff
@@ -0,0 +1,579 @@
+---
+layout: post
+title: Error Bar Chart in .NET MAUI Chart Control | Syncfusion
+description: Learn here all about error bar chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Error Bar in .NET MAUI Chart
+
+[ErrorBarSeries]() indicates the errors or uncertainty in reported values. This will find the possible variations in measurements, and in Chart control these values are displayed as data points.
+The [HorizontalErrorValue]() and the [VerticalErrorValue]() is used to set the error value(variation) to the series.
+
+N> The Error Bar is not an individual Chart, it associate with a main Chart. Here, we use Scatter Series Chart as Main Chart with the Error Bar Series Chart Support
+
+The following code examples illustrates how to create error bar series:
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+
+    <chart:SfCartesianChart.XAxes>
+        <chart:NumericalAxis />
+    </chart:SfCartesianChart.XAxes>
+
+    <chart:SfCartesianChart.YAxes>
+        <chart:NumericalAxis />
+    </chart:SfCartesianChart.YAxes>   
+
+     </chart:SfCartesianChart.YAxes>
+            <chart:ScatterSeries ItemsSource="{Binding EnergyProductions}" 
+                                 XBindingPath="ID" 
+                                 YBindingPath="Coal"
+                                 PointHeight="20"
+                                 PointWidth="20">
+            </chart:ScatterSeries>
+            <chart:ErrorBarSeries ItemsSource="{Binding EnergyProductions}"
+                                  XBindingPath="ID"
+                                  YBindingPath="Coal"
+                                  VerticalErrorValue="50"
+                                  HorizontalErrorValue="0.5">
+            </chart:ErrorBarSeries>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    NumericalAxis primaryAxis = new NumericalAxis();
+    chart.XAxes.Add(primaryAxis);
+    NumericalAxis secondaryAxis = new NumericalAxis();
+    chart.YAxes.Add(secondaryAxis);
+
+    ScatterSeries series = new ScatterSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        PointWidth = 20,
+        PointHeight = 20
+    };
+
+    ErrorBarSeries errorBar = new ErrorBarSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50
+    };
+
+    chart.Series.Add(series);
+    chart.Series.Add(errorBar);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Error Bar Series in MAUI Chart](Chart-types_images/BasicRenderingErrorBar.png)
```

Comment:
image name change like 

MAUI_ErrorBar_featurname

Change this for all images.

Reply:
I replace image names